Fibonacci_Number/Fibonacci_Number.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+using namespace std;
+
+// Function to print nth fibonacci number
+int FibonacciNumber (int num) 
+{ 
+    if (num <= 1) 
+        return num; 
+    return FibonacciNumber(num - 1) + FibonacciNumber(num - 2); 
+} 
+
+// Function to print fibonacci series  
+int FibonacciSeries (int num)  
+{  
+    int a = 0, b = 1, i;  
+
+    if (num < 1)  
+        return 0;  
+
+    for (i = 1; i <= num; i++)  
+    {  
+        cout << b << " ";  
```

Comment:
Remove space around ``<<`` & ``>>`` in cout & cin statements.

Reply:
But in other pr, mentors told me to add space around <<.

Reply:
https://www.codesdope.com/cpp-gear-up/

Reply:
Also nowhere mentioned in coding conventions (https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141388.html)

Reply:
Done please review.